src/address_book.test.js

```diff
@@ -35,17 +35,21 @@ describe('Address book', () => {
   const filePath = path.relative(process.cwd(), path.join(__dirname, '/../utils/files/address_book_test.csv'))
   const ENSName = 'francotest.eth'
 
-  test('Addrress book', async (done) => {
-    console.log('Addrress book')
+  test('Address book', async (done) => {
+    console.log('Address book')
+    console.log('Loads safe form, giving name to the safe and the first 2 owners')
+    console.log('Enter into address book. Validates 3 entries present by name (the load safe process created them)')
```

Comment:
Does it do all three things at the same time?

Reply:
Yeah actually the second comment we can erase it since it's the same behaviour for every tests that loads a safe